### PR TITLE
Update sane-doc-reports

### PR DESCRIPTION
**Pull request by agamm**
## Status
Ready

## Related
related: lhttps://github.com/demisto/etc/issues/18250
https://github.com/demisto/sane-doc-reports/pull/68

## Description
Updates to the latest sane-doc-reports

### CODE REVIEW

**Comment by meirwah** [comment]
@agamm what is status here ?

**Comment by agamm** [comment]
> @agamm what is status here ?

On hold until @hirshbergshachar restores this as a priority to finish 👍

**Comment by glicht** [comment]
Image size has gone up to 450MB+ please review

**Comment by meirwah** [comment]
@glicht I see the base image changed to `demisto/python3-deb` , what is avg size for other images using this as base ?

**Comment by glicht** [comment]
> @glicht I see the base image changed to `demisto/python3-deb` , what is avg size for other images using this as base ?

It depends what you install. The base image is around 60MB. I think chromium is our largest one and it is at ~240MB. https://github.com/demisto/dockerfiles/blob/repository-info/demisto/chromium/last.md .

**Comment by meirwah** [comment]
@agamm which libs you added in this PR ?

**Comment by agamm** [comment]
@glicht Is it better now? (also user-wise?)

File: docker/sane-doc-reports/Dockerfile [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 
-RUN apk add --no-cache --virtual .build-deps gcc musl-dev \
- && pip install cython \
- && apk del .build-deps
-RUN apk add --update --no-cache g++ gcc libxslt-dev freetype-dev jpeg-dev zlib-dev tzdata
-RUN apk --update add --no-cache --virtual .build-dependencies build-base wget git \
-  && pip install --no-cache-dir -r requirements.txt \
-  && apk del .build-dependencies
-RUN ln -fs /usr/share/zoneinfo/UTC /etc/localtime
+RUN npm install svgexport -g --unsafe-perm=true
```
Comment (glicht) [inline docker/sane-doc-reports/Dockerfile:11]: You should install a specific version using `svgexport@<version>` or use a package-lock.json file (I see you have one already).

**Pull request merged**